pyproject.toml

```diff
@@ -1,3 +1,15 @@
 [tool.black]
 line-length = 119
-target_version = ['py37']
\ No newline at end of file
+target_version = ['py37']
+
+[tool.ruff]
+# Ignored rules:
+#   "E501" -> line length violation
+#   "F821" -> undefined named in type annotation (e.g. Literal["something"])
+ignore = ["E501", "F821"]
```

Comment:
Since `setup.cfg` was deleted, what happens to the unused ignore rules that were defined there? [E203](https://www.flake8rules.com/rules/E203.html) and [W503](https://www.flake8rules.com/rules/W503.html)

Reply:
True, I'll check this later, nice catch!

Reply:
honestly not too picky, so if we want to fix this later I'm ok with it. If you look into it and its too time consuming I'm fine with moving to separate PR later to patch

Reply:
don't wanna block this PR much longer

Reply:
I'll have a look in a couple hours whenever I'm out of office, and then we can merge :) Thanks